lucene/spatial3d/src/test/org/apache/lucene/spatial3d/TestGeo3DPoint.java

```diff
@@ -1328,6 +1329,32 @@ public void testEncodeDecodeIsStable() throws Exception {
     }
   }
 
+  public void testDistanceQuery() throws Exception {
+    Directory dir = newDirectory();
+    RandomIndexWriter w = new RandomIndexWriter(random(), dir);
+
+    PlanetModel planetModel = PlanetModel.WGS84;
```

Comment:
Can we test this with the PlanetModel randomly selected from all three (Clarke, Sphere, WGS84)?

Reply:
done